examples/hello_world/android/app/build.gradle

```diff
@@ -2,6 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
```

Comment:
Similarly, interesting that this line is needed when there wasn't an `apply plugin: "kotlin-android"` before.

Reply:
It's actually a side effect of me copy pasting these lines from my other project. It's not needed. Thanks for the catch.